api/release-notes/release-notes-202311.md

```diff
@@ -0,0 +1,45 @@
+---
+sidebar_position: 2
+---
+
+# iCure API version 202311.0.0
+
+This iCure release is a long term support release. It is going to be available at https://v202311.api.icure.cloud for at least 30 months.
```

Comment:
A start date and an end date for the LTS support could be great since we don't see directly the creation date of this document